.Rprofile

```diff
@@ -1,6 +1,5 @@
 source("renv/activate.R")
 if (Sys.getenv("GITHUB_ACTIONS") == "") {
-  envsetup::rprofile(config::get(file = "_envsetup.yml", config = "prod"))
```

Comment:
Thanks for removing this @bms63 .  The adam and tlf scripts still work without envsetup.